cmd/root.go

```diff
@@ -60,7 +60,8 @@ You can also explicitly specify the scheme to use:
     {{.appName}} dir:path/to/yourproject                read directly from a path on disk (any directory)
     {{.appName}} sbom:path/to/syft.json                 read Syft JSON from path on disk
     {{.appName}} registry:yourrepo/yourimage:tag        pull image directly from a registry (no container runtime required)
-
+    {{.appName}} att:attestation.json --key cosign.pub  extract and scan SBOM from attestation file. Checkout Syft to attest your SBOMs
+    {{.appName}} attestation.json --key cosign.pub			the scheme att: is optional
```

Comment:
nit: instead of informing the user that the scheme is optional I would use the above help as an example and say something like `explicitly use the input as an attestation`